src/main/java/user_register/frameworks_and_drivers/LoggedInScreen.java

```diff
@@ -0,0 +1,74 @@
+package user_register.frameworks_and_drivers;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+// Frameworks/Drivers layer
+
+public class LoggedInScreen extends JFrame implements ActionListener {
+    /**
+     * The username chosen by the user
+     */
+    JTextField username = new JTextField(15);
+    /**
+     * A window with a title and a JButton.
+     */
+    public LoggedInScreen() {
+
+        JLabel title = new JLabel("Logged-in Screen");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JButton logOut = new JButton("Log out");
+        JButton profile = new JButton("Profile");
+        JButton chats = new JButton("Chats");
+        JButton friends = new JButton("Friends");
+        JButton groups = new JButton("Groups");
+//        JButton changePassword = new JButton("Change password");
```

Comment:
Since the reporting system is opened through the Logged in screen, plz add a "report" button to connect to my code.